staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -28,6 +28,7 @@ const (
 	StatefulSetRevisionLabel       = ControllerRevisionHashLabelKey
 	DeprecatedRollbackTo           = "deprecated.deployment.rollback.to"
 	DeprecatedTemplateGeneration   = "deprecated.daemonset.template.generation"
+	StatefulSetPodNameLabel        = "statefulset-pod-name"
```

Comment:
This should be prefixed.  We have absolutlely no consistency across teh codebase but it should be either:

statefulset.kubernetes.io/pod-name 

or

kubernetes.io/statefulset-pod-name 

I lean towards the former.

revision hash should probably be, also, but it's harder to fix retroactively.